project/AkkaBuild.scala

```diff
@@ -20,13 +20,15 @@ object AkkaBuild {
 
   lazy val buildSettings = Dependencies.Versions ++ Seq(
     organization := "com.typesafe.akka",
-    version := "2.5-SNAPSHOT")
+    version := (version in ThisBuild).value)
```

Comment:
could you document this a bit more? It seems rather magical where the actual value comes from now

Reply:
ping @2m comment and merge? :)

Reply:
Okay. But with sbt `inspect` is the king. It shows where values for settings come from more robustly than comments.

Reply:
document intent why we do this so weirdly, and hint at inspect then

Reply:
Added a comment